src/main/resources/templates/gardens/add.html

```diff
@@ -41,14 +41,15 @@
         <div class="form-group">
             <label th:for="owner">Garden Owner: </label>
             <select name="owner">
-                <option disabled selected value> -- select an option -- </option>
```

Comment:
I had to take out this line to get validation to work. It automatically selects the first owner in the drop down, so can't leave it empty.

Reply:
Per offline conversation with the team, we determined that this should be added back because it seems to work with the drop down